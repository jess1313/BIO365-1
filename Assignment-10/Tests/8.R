suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("8.R"))))

freqData1 = cbind(0:14, c(265, 49, 21, 19, 10, 10, 2, 2, 4, 2, 1, 4, 3, 1, 2))
freqData2 = cbind(1:10, c(42, 1000, 12, 1, 819, 29, 292, 737, 291, 2))

print(round(calcMeanFreqData(freqData1), 8))
print(round(calcMeanFreqData(freqData2), 8))