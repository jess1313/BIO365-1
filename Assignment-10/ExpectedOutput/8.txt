[1] 1.126582
[1] 5.249612
